Python/ceval.c

```diff
@@ -506,7 +506,9 @@ _PyEval_MatchClass(PyThreadState *tstate, PyObject *subject, PyObject *type,
         }
         if (match_self) {
             // Easy. Copy the subject itself, and move on to kwargs.
-            PyList_Append(attrs, subject);
+            if (PyList_Append(attrs, subject) == -1) {
```

Comment:
```suggestion
            if (PyList_Append(attrs, subject) < 0) {
```
I think this is more idiomatic

Reply:
Never mind, either is fine.